brambling/forms/organizer.py

```diff
@@ -265,6 +266,30 @@ def _post_clean(self):
             self._update_errors(e)
 
 
+class CustomFormForm(forms.ModelForm):
+    class Meta:
+        model = CustomForm
+        exclude = ('event',)
```

Comment:
I think the UI for multiple forms needs to be pretty heavily rethought and maybe even the data structures. It doesn't make sense to have both attendee and event forms in the same table if you're going to be reordering them (or even thinking about their order) in that view.